From b5d6a388b6b9d7b5b17e7cd2d468986d71b13f42 Mon Sep 17 00:00:00 2001
From: Bhargav Shah <bhargavshah1988@gmail.com>
Date: Sat, 22 Jun 2019 12:27:16 +0530
Subject: [PATCH 20/21] riscv: sifive: fu540: Enable SiFive SPI and MMC SPI
 drivers

This patch enables SiFive SPI and MMC SPI drivers for the
SiFive Unleashed board.

Signed-off-by: Bhargav Shah <bhargavshah1988@gmail.com>
Signed-off-by: Anup Patel <anup.patel@wdc.com>
Upstream-Status: Submitted
---
 board/sifive/fu540/Kconfig | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/board/sifive/fu540/Kconfig b/board/sifive/fu540/Kconfig
index f46437901d..d6d5c7d170 100644
--- a/board/sifive/fu540/Kconfig
+++ b/board/sifive/fu540/Kconfig
@@ -38,6 +38,14 @@ config BOARD_SPECIFIC_OPTIONS # dummy
 	imply PHY_LIB
 	imply PHY_MSCC
 	imply SIFIVE_SERIAL
+	imply SPI
+	imply DM_SPI
+	imply SIFIVE_SPI
+	imply MMC
+	imply DM_MMC
+	imply MMC_SPI
+	imply MMC_BROKEN_CD
+	imply CMD_MMC
 	imply SMP
 
 endif
-- 
2.22.0

